components/theme-toggler.tsx

```
@@ -24,56 +24,61 @@ export function ThemeToggler({ compact = false }: ThemeTogglerProps) {
           className="relative flex items-center gap-1.5 px-3 py-1.5 text-sm bg-gradient-to-r from-gray-100 to-gray-200 dark:from-gray-900/95 dark:to-gray-800/95 backdrop-blur-sm text-gray-800 dark:text-white rounded-lg hover:from-gray-200 hover:to-gray-300 dark:hover:from-gray-800/95 dark:hover:to-gray-700/95 transition-all duration-300 border border-gray-100 dark:border-gray-700/50 hover:border-gray-400 dark:hover:border-gray-600/70 group overflow-hidden"
-          aria-label={`Current theme: ${theme || 'loading'}`}
+          aria-label={`Current theme: ${theme || "loading"}`}
         >
-          <div className={`absolute inset-0 opacity-0 group-hover:opacity-100 transition-opacity duration-300 ${
-            theme === "light" ? "bg-yellow-500/5" : "bg-blue-500/5"
-          }`}></div>
+          <div
+            className={`absolute inset-0 opacity-0 group-hover:opacity-100 transition-opacity duration-300 ${
+              theme === "light" ? "bg-yellow-500/5" : "bg-blue-500/5"
+            }`}
+          ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)